recipes/eigen/config.yml

```diff
@@ -0,0 +1,7 @@
+versions:
+    "3.3.5":
+      folder: all
+    "3.3.6":
```

Comment:
I think we can have just the latest version as the previous ones are patches

Reply:
Since it is a headers only library and the packages are really small I decided to add the last three versions just in case somebody needs them, but I don't mind removing them if that's what the majority here think is the right thing to do.

Reply:
Yes please, keep only the latest version